web/actions/form_submissions_retriever.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Formerer.FormSubmissionsRetriever do
+  import Ecto.Query
+  alias Formerer.Submission
```

Comment:
I'm a big fan of grouping multiple aliases together with `alias Formerer.{a,b}` style syntax and I've tried to keep to it through the project